src/style.css

```diff
@@ -1,3 +1,68 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  align-items: center;
+  width: 100%;
+  padding: 0 50px;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  display: flex;
+  justify-content: space-between;
+}
+
+.nav {
+  display: flex;
+}
+
+.nav_list {
+  list-style: none;
+  padding: 0;
+  display: flex;
+  align-items: center;
+}
+
+.nav_item {
+  margin-left: 20px;
+}
+
+.nav_link {
+  display: block;
+  position: relative;
+  color: black;
+  text-decoration: none;
+  text-align: center;
+  line-height: 60px;
+}
+
+.nav_list :first-child {
```

Comment:
The selector `.nav_list :first-child` has a space before `:first-child`, which might cause it to select the first child of any element inside `.nav_list`, not just the first `.nav_item`. Consider removing the space to target the first `.nav_item` directly: `.nav_list:first-child`.